libs/shared/src/consts/providers/provider.enum.ts

```diff
@@ -55,6 +61,7 @@ export enum PushProviderIdEnum {
   FCM = 'fcm',
   APNS = 'apns',
   EXPO = 'expo',
+  Pushwoosh = 'pushwoosh',
```

Comment:
```suggestion
  PUSHWOOSH = 'pushwoosh',
```
Please keep the consistency by following the Enum standard.

Reply:
This was a bad suggestion. I am reverting it back. 😓